measCompApp/src/Makefile

```diff
@@ -60,6 +60,7 @@ measCompApp_LIBS += sscan
 measCompApp_LIBS += autosave
 measCompApp_LIBS += asyn
 measCompApp_LIBS += seq pv
+measCompApp_LIBS += qsrv pvAccessIOC pvAccessCA pvAccess pvData
```

Comment:
There is no need to build this IOC with PV access.  If you want to build you real IOC with them that is fine.  Please revert to master.

Reply:
ok, commented out this line